core/src/main/java/edu/mines/jtk/mosaic/TileAxis.java

```diff
@@ -615,7 +627,14 @@ boolean updateAxisTics() {
     // Compute new axis tics constructed with the best-fit tic interval 
     // computed above, but now for the currently visible range [v0,v1] 
     // of world coordinates. These axis tics are painted by this axis.
-    _axisTics = new AxisTics(v0,v1,dtic);
+    if(p.getScale() == Scale.LINEAR || p.getScale() == Scale.AUTO)
```

Comment:
Regarding the Scale.AUTO - I'm curious if there's an efficient way to guess whether the data set were logarithmic. Just curious